text_mining.py

```diff
@@ -0,0 +1,82 @@
+def tweet_list(hashtag, number):
+    """
+    Takes in a string and value find and returns a list of tweets the value
+    long that contain the string
+    """
+    from pattern.web    import Twitter
```

Comment:
It's typically best practices to import outside of functions, especially if the function may be called more than once.